### PR TITLE
2017 fellows bios - OWF

### PR DESCRIPTION
**Pull request by auremoser**
I added
* photos for 2017 fellows
* bios for 2017 fellows
* new page under `open-web-fellows/fellows2017`

Here's a local version:

screencapture-127-0-0-1-8080-en-us-open-web-fellows-fellows2017-1504921917524

Merge when ready, thanks so much!

### CODE REVIEW

**Comment by auremoser** [comment]
PS. I used last year's header image, maybe we want to make a new asset for that. cc/ @hannahkane @sabrinang ?

**Comment by hannahkane** [comment]
Thanks, @auremoser!

I think we can keep the same header image. @sabrinang is pretty booked up with Mozfest work atm.

File: src/pages/open-web-fellows/fellows2017.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+              companyHref="https://www.hrw.org/"
+              handle="baricks"
+              handleHref="https://twitter.com/baricks"
+              link="http://beccaricks.com"
+              linkName="Rebecca's website"
+            >
+              Rebecca is a multidisciplinary software developer, researcher, and advocate who is engaged in issues of privacy and freedom of expression online. Drawing on a background in Middle East politics and Arabic, she has conducted research at New York University on algorithmic and surveillance systems, with a particular emphasis on government repression. A self-taught programmer, she is passionate about building digital tools for activists that promote free speech on the internet.
+            </FellowBlock>
+          </ContentContainer>
+          <ContentContainer className="grey slant">
```
Comment (ScottDowne) [inline src/pages/open-web-fellows/fellows2017.js]: `ContentContainer className="grey slant"` these classnames should probably be staggered to remove the weirdness in the last 5 fellow bios.

**Comment by ScottDowne** [comment]
Just the one comment, otherwise looks great to me. 🚀

**Comment by auremoser** [comment]
Thank you @ScottDowne ! I didn't even notice the class and thought it was some random funky design thing. It actually looked kind of cool, messed up, but I just corrected it for consistency.

Ready to merge if there are no other changes, we'll be announcing the fellows tomorrow AM so deploying before 9:00AM EST would be lovely! Thank you!

2017-fellows-2

**Comment by ScottDowne** [comment]
Looks good to me.

Is this something we can ship now, or is it waiting on anything?

**Comment by auremoser** [comment]
Feel free to :shipit: !

**Comment by TheoChevalier** [comment]
@auremoser Hey Aurélia, in case you know whom to ask, just a heads up that in https://blog.mozilla.org/blog/2017/09/13/mozilla-announces-15-new-fellows-science-advocacy-media/ the link “Learn more about our Open Web Fellows.” points to https://advocacy.mozilla.org/en-US/open-web-fellows/fellows2016 instead of https://advocacy.mozilla.org/en-US/open-web-fellows/fellows2017

**Comment by auremoser** [comment]
@TheoChevalier Hey théo! Thank you for that bug, I believe @KevZawacki will get on the update when he can, for now, hopefully folks can figure out that it's a different tab. Apologies for the delay, and thank you so much for flagging!

**Comment by KevZawacki** [comment]
Thanks for the heads up, @theo & @auremoser. I've just updated the link.

**Pull request merged**